docs/product-assembly_running_employee_rostering_in_wb/src/main/asciidoc/wb-employee-rostering-shift-required-skills-met-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='wb-employee-rostering-shift-required-skills-met-proc']
+= Create the `ShiftReqiredSkillsAreMet` guided rule 
+
+The `ShiftReqiredSkillsAreMet` guided rule creates a hard constraint that ensures all shifts are assigned an employee with the correct set of skills. In the employee rostering example, this constraint is created using the guided rule editor. 
+
+image::ShiftRequiredSkillsAreMet.png[ShiftRequiredSkillsAreMet Guided Rule]
+
+.Procedure
+. Go to *Menu* -> *Design* -> *Projects* and click the project name.
+. Click *Create New Asset* -> *Guided Rule*.
+. Enter `ShiftReqiredSkillsAreMet` as the *Guided Rule* name and select the `employeerostering.employeerostering` *Package*. 
+. Click *Ok* to create the rule asset.
+. Add a *WHEN* condition by clicking the image:5686.png[] in the *WHEN* field.
+. Select `ShiftAssignment` in the *Add a condition to the rule...* window. Click *+Ok*. 
+. Click on the `ShiftAssignment` condition, and select `employee` from the *Add a restriction on a field* drop down list.
+.. In the editor, click the drop down list next to `employee` and select `is not null`.
+. Click on the `ShiftAssignment` condition, and click *Expression editor*.
+.. In the editor, click `[not bound]` to open the *Expression editor*, and bind the expression to the variable `$requiredSkill`. Click *Set*.
+.. In the editor, next to `$requiredSkill`, select `shift` from the first drop down list, then `requiredSkill` from the next drop down list.
+. Click on the `ShiftAssignment` condition, and click *Expression editor*.
+.. In the editor, next to `[not bound]`, select `employee` from the first drop down list, then `skills` from the next drop down list.
+.. Leave the next drop down list as `Choose...`.
+.. In the next drop down box, change `--please choose--` to `excludes`.
```

Comment:
IMO, I think "change **--please choose--**" is in the same camp as final ellipses in the UI, which are to be deleted (per my previous comment). For that reason, I've just said "change **please choose**" and the like in my own docs, for whatever it's worth. Again, super minor and totally your call.